src/main/modules/pcq/service.ts

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import axios from 'axios';
+
+import { PCQProvider } from './index';
+
+export class PcqService {
+  async getPcqHealthStatus(url: string): Promise<string | undefined> {
+    try {
+      const response = await axios.get(url);
+      return response.data.status;
+    } catch (err) {
+      PCQProvider.log('error', err);
+      return '';
```

Comment:
why is the error not handled to comply async fn?

Reply:
We dont have to throw error, We just need to log it, added promise resolve to send down status when error